sky/engine/core/dart/painting.dart

```diff
@@ -179,8 +180,13 @@ enum VertexMode {
   triangleFan,
 }
 
+/// An interface for recording drawing commands.
```

Comment:
Mention how to make use of a canvas you created yourself, namely, that you have to have a picture recorder, that you have to endRecording on the picture recorder, and what you can then do with the canvas (put it in a scene builder or a canvas).